pkg/openstack/version.go

```diff
@@ -202,3 +202,47 @@ func ReconcileVersion(ctx context.Context, instance *corev1beta1.OpenStackContro
 
 	return ctrl.Result{}, version, nil
 }
+
+func compareStringPointers(a, b *string) bool {
```

Comment:
I would still rename this accoridng to https://github.com/openstack-k8s-operators/openstack-operator/pull/712/files#r1528375598

Reply:
I added a commit for this into the PR